tests/phpunit/elementor/modules/atomic-widgets/styles/__snapshots__/Test_Styles_Renderer__test_render__style_with_nested_background_image_transformers__1.txt

```diff
@@ -1 +1 @@
-.test-style{background:linear-gradient(blue, blue),url(" https://example.com/image.jpg ") 0% 0% / cover red;}
```

Comment:
Maybe check with Omer if there is a way to update the url according to the resolution.